mission_control/navigator_missions/navigator_singleton/navigator.py

```diff
@@ -24,21 +24,35 @@
 
 
 class MissionResult(object):
-    def __init__(self, success=True, response="", need_rerun=False, post_function=None):
+    '''
+        The response can be used by the mission runner to determine why a missions failed (assuming it's not a normal
+            runtime issue). Add whatever response types you want.
+        Also with this class you can tell the mission runner whether you want to rerun the mission (if it failed), and
+            give an optional function to run after the mission completes (regardless of pass/fail).
+        Finally, this provides a good way to print out mission results.
+    '''
```

Comment:
These are supposed to be double quotes.
The indentation should match the quotes
The first line needs to be Imparative and end with a period.
There should be a space between first and second line